assignments/function-conversion.js

```diff
@@ -1,24 +1,32 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+//  let myFunction = function () {
+//  console.log("Function was invoked!");
+//  };
+//  myFunction();
+const myFunction = () => "Function was invoked!";
+console.log(myFunction());
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+anotherFunction = (param) => "Example";
+console.log(anotherFunction());
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+add = (param1, param2) => param1 + param2;
+console.log(add(1,2));
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+subtract = (param1, param2) => param1 - param2;
+console.log(subtract(2,1));
```

Comment:
a variable declaration should start with a keyword, let, const..., e.g this should be `const subtract = {...}`